src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -53,6 +55,10 @@ internal static class RemoteHostOptions
         public static readonly Option<int> MaxPoolConnection = new Option<int>(
             nameof(InternalFeatureOnOffOptions), nameof(MaxPoolConnection), defaultValue: 15,
             storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + nameof(MaxPoolConnection)));
+
+        public static bool IsServiceHubProcess64Bit(Workspace workspace)
```

Comment:
`Use64BitServiceHubProcess` ?

Reply:
I'll follow up with refactoring later and rename it then.